src/data/type/user-output.type.ts

```diff
@@ -1,8 +1,9 @@
 import { ObjectType, Field, ID } from 'type-graphql';
+import { UserEntity } from '../entity/user.entity';
 
 @ObjectType()
-export class UserOutputType {
-  @Field((type) => ID)
+export class UserOutputType implements Partial<UserEntity> {
+  @Field(() => ID)
```

Comment:
O tipo ID geralmente é interpretado pelo GraphQL como string, sendo bastante usado quando os ids são uuids ou algum outro identificador único similar. Mas quando o id é um número inteiro, será que faz sentido usar ID? Será que Int não seria mais interessante? 🤔

Reply:
@andrewmsilva  acho que justamente por isso o ID faz sentido, ele manda sempre string pro graphql, então se por algum motivo o id, internamente, for alterado de string pra number ou o oposto, whatever, para quem consome a api não muda nada.